app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3:
+        return
+
+    mv, filename, destination_path = parts
+    new_file_name = ""
+
+    if mv == "mv" and os.path.exists(filename):
+        if destination_path.endswith("/"):
```

Comment:
You could use `os.path.isdir` to check wether the destination is a directory of not. Omit using `/` to perform this check.

Reply:
Thanks